src/_data/catalog/widgets.json

```diff
@@ -1657,6 +1921,28 @@
     "link": "https://api.flutter.dev/flutter/material/Switch-class.html",
     "image": "<img alt='' src='/assets/images/docs/widget-catalog/material-switch.png'>"
   },
+  {
+    "name": "Switch",
+    "description": "Switches toggle the state of a single item on or off.",
```

Comment:
**issues:**
- Make consistent with other descriptions.
- Remove the circular definition.
```suggestion
    "description": "Toggle that changes the state of a single item to on or off.",
```